pkgs/applications/networking/browsers/chromium/update.py

```diff
@@ -6,15 +6,20 @@
 import subprocess
 import sys
 
+from base64 import b64decode
 from codecs import iterdecode
 from collections import OrderedDict
 from os.path import abspath, dirname
-from urllib.request import urlopen
+from requests import get
```

Comment:
Do we really need to switch to `requests` here (for the `.text` below)? It's not part of the Python standard library so at least we'd have to specify it as a dependency (2nd line, if necessary).